dags/merino_jobs.py

```diff
@@ -72,14 +73,37 @@ def merino_job(
             "--gcs-path",
             "moz-fx-data-prod-external-data/contextual-services/merino-jobs/wikipedia-exports",
             "--gcp-project",
-            "moz-fx-data-shared-prod",
+            "moz-fx-data-shared-nonprod",
         ],
         env_vars={
-            "MERINO_ENV": "production",
+            "MERINO_ENV": "staging",
```

Comment:
I think this can also stay using 'production'. This environment variable tells Merino how to layer its config files ([defined here](https://github.com/mozilla-services/merino-py/tree/main/merino/configs)) which in turn configures the logging to use the mozlog logging format. I believe we use `production` to mean anything running out of dev or testing